src/parsers/unix_socket.cpp

```diff
@@ -66,6 +66,7 @@ unix_socket parse_unix_socket_line(const std::string& line)
     // ffff8db2fd23a000: 00000003 00000000 00000000 0001 03 17031 /run/systemd/journal/stdout
     // ffff8db2f696e000: 00000003 00000000 00000000 0001 03 15699 /run/systemd/journal/stdout
     // ffff8db2f3e09400: 00000002 00000000 00000000 0002 01 21401
+    // cad3ab00:         00000003 00000000 00000000 0001 03 7242 /var/qmux_client_socket    401
```

Comment:
Can we please add a unit-test for this case? (`test/test_unix_socket.cpp`)
This should do the trick:
```
    SECTION("Qmux socket")
    {
        size_t skbuff = 0xcad3ab00;

        line << skbuff << ":         00000003 00000000 00000000 0001 03 "
                "7242 /var/qmux_client_socket    401";

        expected.skbuff       = skbuff;
        expected.ref_count    = 0x00000003;
        expected.protocol     = 0x00000000;
        expected.flags        = 0x00000000;
        expected.socket_type  = pfs::unix_socket::type::stream;
        expected.socket_state = pfs::unix_socket::state::connected;
        expected.inode        = 7242;
        expected.path         = "/var/qmux_client_socket    401";
    }
```